array/unique.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "unique",
-  "description": "Returns only unique elements of the given array.",
+  "description": "Returns unique elements of the given array.",
   "signature": "(xs: any[]) => any[]",
   "examples": [
     {
       "language": "javascript",
-      "content": "unique(); // ⇒ TODO"
+      "content": "unique([1, 2, 3, 4, 3, 4, 3, 6]);\n// ⇒ [1, 2, 3, 4, 6]"
     }
   ],
-  "questions": ["TODO: List questions that may this function answers."]
+  "questions": ["How to get all unique values in an array?"]
```

Comment:
Again, I'm not sure if `get ... in` is proper verb for saying that we take something out of array. https://www.oxfordlearnersdictionaries.com/definition/english/get-in

Reply:
I have changed it to "How to find...".